platform/functions/remix-server/regional-server.mjs

```diff
@@ -107,51 +44,59 @@ const createApigHandler = (build) => {
     context.callbackWaitsForEmptyEventLoop = false;
     const request = convertApigRequestToNode(event);
     const response = await requestHandler(request);
```

Comment:
This is the line of code where lots of requests get stuck at some point... Not sure if changes in the handler can fix this.